### PR TITLE
Fix the bottom gap on the transaction list scene

### PR DESCRIPTION
**Pull request by swansontec**
Also remove the second copy, so the scene only exists inside the tab navigator.

### CHANGELOG

 none

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204340335884555
  - https://app.asana.com/0/0/1204343630586365

### CODE REVIEW

File: src/components/Main.tsx
Comment (samholmes) [inline src/components/Main.tsx:1]: This should be in a separate commit, but that's just a nit that I'm not necessarily going to push for.

File: src/components/scenes/GuiPluginViewScene.tsx [hunk 1]
```diff
@@ -190,7 +190,7 @@ export function GuiPluginViewScene(props: Props): JSX.Element {
       : 'Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_1 like Mac OS X) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.0 Mobile/14E304 Safari/602.1'
 
   return (
-    <SceneWrapper background="theme">
+    <SceneWrapper background="theme" hasTabs={route.name !== 'pluginView'}>
```
Comment (samholmes) [inline src/components/scenes/GuiPluginViewScene.tsx:193]: Didn't I do this for this scene from a previous review? I was confident that I added this as a fixup from review feedback. What happened there in retrospect, I wonder.

**Review by samholmes — APPROVED** [review]
Looks good to me.

File: src/components/scenes/GuiPluginViewScene.tsx (hunk 1 above)
Comment (swansontec) [reply src/components/scenes/GuiPluginViewScene.tsx:193]: Yes, you did do it. However, it got lost during the EdgeProvider re-write.
